parts/linux/cloud-init/artifacts/cse_benchmark_functions.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+script_start_stopwatch=$(date +%s)
+section_start_stopwatch=$(date +%s)
+SCRIPT_NAME=$(basename $0 .sh)
+SCRIPT_NAME="${SCRIPT_NAME//-/_}"
+declare -A benchmarks=()
+declare -a benchmarks_order=()
+
+check_array_size() {
+  declare -n array_name=$1
+  local array_size=${#array_name[@]}
+  if [[ ${array_size} -gt 0 ]]; then
+    last_index=$(( ${#array_name[@]} - 1 ))
+  else
+    return 1
+  fi
+}
+
+installJq() {
```

Comment:
this is only used in `/vhdbuilder/packer/pre-install-dependencies.sh` we should keep it in `cse_helper.sh` I would say